conjure-java-jaxrs-client/build.gradle

```diff
@@ -29,7 +29,12 @@ dependencies {
     testImplementation project(":conjure-java-retrofit2-client")
     testImplementation project(":conjure-java-jersey-server")
     testImplementation project(':keystores')
-    testImplementation "com.palantir.dialogue:dialogue-serde"
+    testImplementation "com.palantir.dialogue:dialogue-serde", {
+        exclude group: 'com.palantir.conjure.java.runtime'
```

Comment:
why do we need this?

Reply:
Circular dependency on this project which breaks GCV

Reply:
You should be able to do:
```
configurations.all {
  resolutionStrategy {
    preferProjectModules()
  }
}
```

Reply:
Any reason we don't include that in GCV by default?